hw/dv/tools/dvsim/testplans/sec_cm_one_hot_testplan.hjson

```diff
@@ -0,0 +1,28 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  testpoints: [
+    {
+      name: prim_one_hot_check
+      desc: ''' Verify that violating one-hot property generates a fatal alert.
+
+            Stimulus:
```

Comment:
Add a line above Stimulus to make it clear:
```
      One-hot encoded signals are captured in `prim_one_hot` module. 
```

Otherwise the stimulus suggests forcing one-hot encoded signals directly (which would then invite more questions).

Reply:
Added, thanks